lib/data/repositories/data_authentication_repository.dart

```diff
@@ -30,7 +30,7 @@ class DataAuthenticationRepository implements AuthenticationRepository {
 
   /// Registers a `User` using a [email] and a [password] by making an API call to the server.
   /// It is asynchronous and can throw an `APIException` if the statusCode is not 200.
-  Future<void> register({@required String email, @required String password}) async {
+  Future<void> register({@required String firstName, String lastName, String email, @required String password}) async {
```

Comment:
Add `@required` to the `lastName` and `email`

Reply:
Changes made